test/testing_utils/checkers.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package test
```

Comment:
I am also thinking of moving this into a completely separate package, e.g. `testing/checkers` so that the `test` only contains actual tests, not the helpers.